Sourcery/Generating/Template/Swift/SwiftTemplate.swift

```diff
@@ -47,11 +47,11 @@ class SwiftTemplate: Template {
 
         let components = templateContent.components(separatedBy: Delimiters.open)
 
-        var sourceFile = [String]()
+        var processedComponents = [String]()
         var commands = [Command]()
 
         let currentLineNumber = {
-            return sourceFile.joined(separator: "").numberOfLineSeparators
+            return processedComponents.joined(separator: "").numberOfLineSeparators + 1
```

Comment:
I'll add a small comment here if you think this is needed 😉 
I was more on the side that the code was self explanatory here 😊 
Here was my rationale: we are computing the `currentLineNumber`, which starts from 1, and to do that we count the `numberOfLineSeparators`, which starts from 0. So the block is just the transformation of a count of lines to a line number, and this transformation is just doing the `+1`.